Source/Core/DolphinWX/ISOProperties.cpp

```diff
@@ -1246,8 +1247,8 @@ void CISOProperties::OnComputeMD5Sum(wxCommandEvent& WXUNUSED (event))
 	u64 read_offset = 0;
 	md5_context ctx;
 
-	File::IOFile file(OpenGameListItem.GetFileName(), "rb");
-	u64 game_size = file.GetSize();
+	std::unique_ptr<DiscIO::IBlobReader> file(DiscIO::CreateBlobReader(OpenGameListItem.GetFileName()));
+	u64 game_size = file->GetDataSize();
```

Comment:
The size is of what the container contains, not the container itself.

It would be possible to use the size for things other than hashing, such as:
* Showing a special error message when a game tries to read out of bounds as opposed to a generic "read failed" message
* Reading a whole volume to copy it to another file (for instance ISO to GCZ, but that's currently implemented in another way IIRC)